wsi/drm/drmbuffer.cpp

```diff
@@ -89,13 +88,14 @@ void DrmBuffer::Initialize(const HwcBuffer& bo) {
     frame_buffer_format_ = format_;
   }
 
-  FrameBufferManager::GetInstance()->RegisterGemHandles(
-      image_.handle_->meta_data_.num_planes_,
-      image_.handle_->meta_data_.gem_handles_);
+  fb_manager_->RegisterGemHandles(image_.handle_->meta_data_.num_planes_,
+                                        image_.handle_->meta_data_.gem_handles_);
 }
 
 void DrmBuffer::InitializeFromNativeHandle(HWCNativeHandle handle,
-                                           ResourceManager* resource_manager) {
+                                           ResourceManager* resource_manager,
+                                           FrameBufferManager *frame_buffer_manager_) {
```

Comment:
frame_buffer_manager_ -> frame_buffer_manager

Reply:
You made a comment previously that you wanted frame_buffer_manager_? Did you only mean you wanted to change only these two instances or all of them to remove the last underscore?

Reply:
variables passed to a function, shouldn't be frame_buffer_manager_ but just frame_buffer_manager. While class member variables coding style is to be  *frame_buffer_manager_;